scripts/member-account-ram-association.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
Looks good. Just to note, this article https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/ recommends the use of `set -Eeuo pipefail`. As stated in the above: `set -e` by itself is far from enough. We can further improve upon the behavior created by set -e by combining it with set -o pipefail.